cfn/collector.template.template

```diff
@@ -0,0 +1,779 @@
+{
```

Comment:
Template file name should be `{{type}}-collector.template`

Reply:
Please also mention somewhere that the template needs to be implemented and is for illustrative purpose only.

Reply:
So that template will be auto populated and renamed by the creation script. I just rename files on a case by case bassi in the script. rather than putting the templating in the file names themselves. I worried that would screw up reading the files somehow (having {{}} in a file name seems weird to me somehow).